lib/ui/src/modules/ui/components/layout/index.js

```diff
@@ -22,6 +22,7 @@ const downPanelStyle = downPanelInRight => ({
   display: 'flex',
   flexDirection: downPanelInRight ? 'row' : 'column',
   alignItems: 'stretch',
+  position: 'absolute',
```

Comment:
This will break scrolling inside the panel, I think?

I'm confused about this element being width & height : 100% AND also have padding..

Can you provide a reproducible case which caused you to find a problem you wanted to fix?